tests/snippets/types_snippet.py

```diff
@@ -50,13 +55,17 @@ class D(B, C): pass
 
 del D
 
-try: # gc sweep is needed here for CPython...
-    import gc; gc.collect()
-except: # ...while RustPython doesn't have `gc` yet.
-    pass
-
-assert B.__subclasses__() == []
-assert C.__subclasses__() == []
+if gc:
+    # gc sweep is needed here for CPython...
+    gc.collect()
+    # ...while RustPython doesn't have `gc` yet. 
```

Comment:
yet...?

Reply:
Not sure, the comment was there already. I just moved it.